core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/Period.kt

```diff
@@ -9,29 +9,31 @@ private const val WEEKS_PER_MONTH = DAYS_PER_YEAR / MONTHS_PER_YEAR / DAYS_PER_W
 /**
  * Represents subscription or [PricingPhase] billing period.
  */
-public expect class Period
+public expect class Period {
+
+    /**
+     * The number of period units.
+     */
+    public val value: Int
+
+    /**
+     * The increment of time that a subscription period is specified in.
+     */
+    public val unit: PeriodUnit
+
+}
 
 /**
  * The unit of time a [Period] is denoted in.
  */
-public expect enum class PeriodUnit {
+public enum class PeriodUnit {
     DAY,
     WEEK,
     MONTH,
     YEAR,
     UNKNOWN,
 }
 
-/**
- * The number of period units.
- */
-public expect val Period.value: Int
-
-/**
- * The increment of time that a subscription period is specified in.
- */
-public expect val Period.unit: PeriodUnit
-
 public val Period.valueInMonths: Double
```

Comment:
NABD but should we move this extension function inside the `Period` as well? Might not be possible if using `expect/actual` but if we're moving away from that, it would mean one less import needed.